### PR TITLE
updating PretrainedModelInfo for ASR SSL models

### PR DESCRIPTION
**Pull request by krishnacpuvvada**
Signed-off-by: Krishna Puvvada <kpuvvada@nvidia.com>

# What does this PR do ?

Updating PretrainedModelInfo for ASR SSL models

**Collection**: ASR

# Changelog 
- Added conformer large, xlarge info in PretrainedModelInfo

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [x] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [x] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Comment by titu1994** [comment]
Are these checkpoints public ?

File: nemo/collections/asr/models/ssl_models.py [hunk 1]
```diff
@@ -50,7 +50,23 @@ def list_available_models(cls) -> Optional[PretrainedModelInfo]:
         Returns:
             List of available pre-trained models.
         """
-        return []
+        results = []
+
+        model = PretrainedModelInfo(
+            pretrained_model_name="ssl_en_conformer_large",
+            description="For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:nemo:ssl_en_conformer_large",
+            location="https://api.ngc.nvidia.com/v2/models/nvidia/nemo/ssl_en_conformer_large/versions/1.10.1/files/ssl_en_conformer_large.nemo",
```
Comment (titu1994) [inline nemo/collections/asr/models/ssl_models.py:58]: What's version 0.10.1?

Comment (krishnacpuvvada) [reply nemo/collections/asr/models/ssl_models.py:58]: do you mean 1.10.1 ? 
1.10.0 - model trained only with contrastive loss
1.10.1 - model trained contrastive + mlm loss. 
we are using versioning to indicate better models of same (dataset, model ) configuration

**Comment by krishnacpuvvada** [comment]
> Are these checkpoints public ?

yes. these are public.

File: nemo/collections/asr/models/ssl_models.py (hunk 1 above)
Comment (titu1994) [reply nemo/collections/asr/models/ssl_models.py:58]: So how do you access the conformer large 1.10.0 model ? You have links only for 1.10.1

**Review by titu1994 — APPROVED** [review]
Looks fine.

File: nemo/collections/asr/models/ssl_models.py (hunk 1 above)
Comment (krishnacpuvvada) [reply nemo/collections/asr/models/ssl_models.py:58]: following the convention of other models, `from_pretrained` gets the "best" conformer large ssl model (which is 1.10.1 in this case). other versions can be downloaded via versions tab in model card.

**Review by sam1373 — APPROVED** [review]

**Pull request merged**